src/langs/zig/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM kassany/bookworm-ziglang:latest AS wheel-part-zig
```

Comment:
Can we set it to a specific version instead of latest? What I experience with other languages is that in newer versions there are sometimes changes that break the wasm compilation

Reply:
Done.